resources/assets/less/bem/contest-user-entry.less

```diff
@@ -0,0 +1,63 @@
+.contest-user-entry {
+  height: 150px;
+  background: #313D40;
+  margin: 5px;
+  width: calc(50% ~'-' 20px);
+  box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.25);
+  transition: 0.2s cubic-bezier(0.68, -0.55, 0.27, 1.55);
+
+  &__filename {
+    font-weight: lighter;
+    font-size: 15px;
+    margin-top: 25px;
+
+    &::before {
```

Comment:
have you remembered why you did this instead of just using `fa fa-whatever` directly?